### ISSUE TITLE
REA_wheels and REA_cog

### ISSUE THREAD

**Issue by Fracking** [900hasse/FS22_REAwheels#25]
Currently, if REA_cog is enabled, the front tires of small tractors (like the Zetor Proxima HS80) would turn faster than they should (even if you turn off 4WD with mods). It seemed worse with manual transmissions and if there was a lot of weight on the front tires / axle e.g. 3T weight or large bale and going like 1 kph. The wheels tend to dig and sink fairly quickly. This isn't very good if you try to move heavy bales on a grass field with ground destruction enabled. It's also visually unpleasing.
After days of fiddling with REA_wheels and REA_cog, I think I found a solution:
I’ve changed:

wheel.SmootheWheelLoad =  (wheel.SmootheWheelLoad * 0.9)+(ActWheelLoad * 0.1)
to
wheel.SmootheWheelLoad =  (wheel.SmootheWheelLoad * 0.9)+(ActWheelLoad * **0.01**)

It seemed to have fixed/lessened the issue. However, tractors seemed to be able to pull heavier loads (I guess it lower the rolling resistance). 
So I added:

if wheel.RollingDirectionSpeed < 3 then
       wheel.SmootheWheelLoad =  (wheel.SmootheWheelLoad*0.9)+(ActWheelLoad * 0.01);
end;

It’s not perfect, since the front wheels still turn a bit faster than they should and the “load” is not as high at low speed. But they don’t dig in while cruising (it’s still possible to dig in with too much wheel spin) and after 3 kph it keeps the normal REA_wheels.

What do y'all think? Am I missing something?

**Comment by DrewCZ** [comment]
The distribution torque between the front and rear axle in the Differentials tag is related to this. Usually, real tractors have default settings for example 40/60. Playing with these values can help. Modern tractors have of course tools for better and variable distribution, which is not available in the game, unfortunately.

**Comment by Fracking** [comment]
True. However, if I disable COG, there is no such issue. COG moves the weight more rearwards, so in theory, yes, the front wheels would tend to spin more. But if there is a lot of weight on the front axle (almost tipping forwards), there would be no wheel spin. Also, the front wheels keep moving even if the rear wheels are already standing still. This can't be normal behavior.
